### PR TITLE
ENCD-5264 fix error on pages missing type

### PR DESCRIPTION
**Pull request by emro**
Some pages do not have an @type, so when I added a class name based on @type (so that I could make some pages, like the summary page with the body map facet, wider), these pages returned an error. Adding a check for @type eliminates this error.

### CODE REVIEW

File: src/encoded/static/components/app.js
Comment (zoldello) [inline src/encoded/static/components/app.js:1250]: className={`container  ${context['@type'] ? context['@type'].join(' ') : ''`} }

Here is an alternate way

**Review by zoldello — DISMISSED** [review]
The code looks good. My alternate way is just me thinking aloud in a sense and it not significantly different from your approach.

**Comment by emro** [comment]
I thought about writing it that way and then I switched partway through, and didn't have a good reason for doing that. I think it shouldn't make much of a difference, but if the way you suggested is more conventional, I'm happy to push an update.

**Comment by zoldello** [comment]
@emro Your call. I would recommend leaving it as-is. "Tomato" and "Tommatooo" are still the same fruit. The code looks fine as-is.

**Review by zoldello — DISMISSED** [review]
I am approving it assuming you want to leave it as-is. If you do decide to change it, I will know from the new commit and return to this PR.

**Review by zoldello — APPROVED** [review]
Looks fine.

**Review by zoldello — APPROVED** [review]
Looks good.

**Comment by zoldello** [comment]
@emro  Please ignore the re-request for code review.

**Pull request merged**
